yard-cucumber.gemspec

```diff
@@ -56,7 +56,7 @@ Gem::Specification.new do |s|
 
   s.add_development_dependency 'rake', '~> 10'
 
-  s.add_dependency 'cucumber', '>= 2.0', '< 8.0'
+  s.add_dependency 'cucumber', '~> 8.0'
```

Comment:
I think this would leave people using Cucumber 7 hanging out to dry.  I'd suggest just changing the `< 8.0` in the original gemspec to `< 9.0`.

Reply:
Done